hs_core/templates/email/group_membership.html

```diff
@@ -1,9 +1,18 @@
 {% load i18n %}
 {% block main %}
     {% if group_owner %}
-        <p>{% trans "Dear " %}{{ group_owner.first_name }}</p>
-        <p>{% trans "User " %}{{ user.first_name }} has requested to join your HydroShare group ({{ group.name }}).</p>
-        <p>Click<a href="{{ request.scheme }}://{{ request.get_host }}/user/{{ user.id }}"> here </a>to find information about the requesting user.</p>
+        <p>{% trans "Dear " %}{{ group_owner.first_name }}{{ group_owner.last_name }},</p>
+        <p>{% trans "User " %}{{ user.first_name }} {{ user.last_name }} 
+            (<a href="{{ request.scheme }}://{{ request.get_host }}/user/{{ user.id }}">{{ request.scheme }}://{{ request.get_host }}/user/{{ user.id }}</a>), 
+            has requested to join your HydroShare group ({{ group.name }}).</p>
+        
```

Comment:
Looking at the template format in issue #3806, it seems the justification text for joining the group needs to go here. Was there a decision not to implement that?

Reply:
Thanks @pkdash. I thought we decided to go without it but @Castronova if you want me to add JUSTIFICATION for group request, I can do so